### DISCUSSION TITLE
<br> instead of <p></p> by pressing enter

### DISCUSSION THREAD

**Discussion by bgrand-ch** [ueberdosis/tiptap#917] [General Questions] [answered]
Is it possible to change the default behavior on pressing `enter` ?

I would like to generate a `<br>` tag instead of `<p></p>` tags.

**Comment by hanspagel** [comment, marked as answer]
That should be possible using the [HardBreak](https://github.com/ueberdosis/tiptap/blob/main/packages/tiptap-extensions/src/nodes/HardBreak.js) extension. Just bind it to 'Enter' instead of 'Mod-Enter'/'Shift-Enter'. Hope that helps! ✌️

**Reply by bgrand-ch** [reply]
Many thanks!

**Reply by Robxxcraft** [reply]
How to do new paragraph with hardbreak instead paragraph only when press enter?
